packages/app-review/android/build.gradle

```diff
@@ -51,6 +51,8 @@ repositories {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation project(':capacitor-android')
+    implementation 'com.google.android.play:review:2.0.2'
+    implementation 'com.google.android.play:review-ktx:2.0.2'
```

Comment:
I wasn't sure if I need -ktx

Reply:
No, this should not be needed because we use Java instead of Kotlin.